### PR TITLE
Port to Python3

### PR DESCRIPTION
**Pull request by Climax777**
This version is compatible with python3 only

### CODE REVIEW

**Comment by Climax777** [comment]
https://github.com/hlamer/python-ws-discovery/issues/4

This does not work on python2 as well. I haven't been able to make it compatible with both. Any ideas?

**Comment by Climax777** [comment]
@hlamer please review now

File: WSDiscovery.py [hunk 1]
```diff
@@ -405,9 +405,9 @@ def addElementWithText(doc, parent, name, ns, value):
 
 def getDocAsString(doc):
     outStr = ""
-    stream = StringIO.StringIO(outStr)
+    stream = io.StringIO(outStr)
     stream.write(doc.toprettyxml())
-    return stream.getvalue()
+    return stream.getvalue().encode('UTF-8')
```
Comment (petri) [inline WSDiscovery.py]: This is actually returning a bytes object now, not unicode as in the original Python2 getDocAsString.

Comment (Climax777) [inline WSDiscovery.py]: It is fixed now. Python3 uses bytes in the socket send functions.

**Comment by petri** [comment]
By the way, would be good to also add Trove classifiers to setup.py for what Python versions this package is for.

**Comment by petri** [comment]
I think I'm out of comments :) So if the stuff works on Python3, I'd say it's good to merge.

**Comment by hlamer** [comment]
Thanks, @Climax777 
I've also changed github project header

**Pull request merged**